test/test_v2/test_server.c

```diff
@@ -104,7 +108,34 @@ static void release_cb(pmix_status_t status, void *cbdata)
     *ptr = 0;
 }
 
-static void fill_seq_ranks_array(size_t nprocs, int base_rank, char **ranks)
+void set_client_argv(test_params *params, char ***argv)
+{
+    pmix_argv_append_nosize(argv, params->binary);
+    pmix_argv_append_nosize(argv, "-n");
+    if (NULL == params->np) {
+        pmix_argv_append_nosize(argv, "1");
+    } else {
+        pmix_argv_append_nosize(argv, params->np);
+    }
+    if( params->verbose ){
+        pmix_argv_append_nosize(argv, "-v");
+    }
+    if (NULL != params->prefix) {
+        pmix_argv_append_nosize(argv, "-o");
+        pmix_argv_append_nosize(argv, params->prefix);
+    }
+    if (params->nonblocking) {
+        pmix_argv_append_nosize(argv, "-nb");
+    }
+    if (params->collect) {
+        pmix_argv_append_nosize(argv, "-c");
+    }
+    if (params->collect_bad) {
+        pmix_argv_append_nosize(argv, "--collect-corrupt");
+    }
+}
+
+static void fill_seq_ranks_array(size_t nprocs, int base_rank, char **ranks, validation_params *v_params)
```

Comment:
What is the purpose of this function?

Reply:
At this point it's cruft, you're right. Will eliminate.

Reply:
Ah, I remember now: set_client_argv() was formerly in its own file named utils.c, which I eliminated because this was all it contained. It is still functional, although 'nonblocking', 'collect', and 'collect_bad' are not in use, at least at present.

Reply:
No, I was still wrong. It was in test_common.c, not utils.c.

Reply:
@artpol84 My point is, that it is still being utilized, and I didn't really change it, I just moved it around. We can perhaps eliminate it later if you don't like it, but there's so much refactoring in this commit now that I'd prefer that we leave it to the next commit.